core/tests/webdriverio_utils/users.js

```diff
@@ -82,16 +82,17 @@ var login = async function(email, useManualNavigation = true) {
   });
 };
 
-var logout = async function() {
+var logout = async function(errorMessage = '') {
```

Comment:
Can you implement this such that you don't need to pass an errorMessage argument into this function? This will allow your code in general.js to be used for any function.

Basically, presumably something in logout() is throwing an error. The idea is that callFunctionAndCollectFullStackTraceOnError() should catch this error, add info to it about the original location of the caller, and throw it again -- right?

Reply:
Yes. Before logout throws an error, webdriverio will print these red logs. And then callFunctionAndCollectFullStackTraceOnError() catch the error print black logs.  That's why red logs haven't changed if we don't pass errorMessage. 
![image](https://github.com/oppia/oppia/assets/85102180/c7fd392e-5900-47e4-9cc5-0dee6114c217)
![image](https://github.com/oppia/oppia/assets/85102180/d8131ad5-d1d7-4599-bcd7-f3c3e1751966)

Reply:
Ah, I see. But that really reduces the generality -- we cannot be adding this argument to all the functions. So if that's what's needed to make the red logs show the full stack then I don't think it's worth it.

Let's prioritize doing something that works in general. You can add more info to the general.js method's docstring to explain where the logs are printed (i.e. in the non-red section) and why we can't put them in the red section.